bigbluebutton-html5/private/config/public/acl.yaml

```diff
@@ -0,0 +1,57 @@
+acl:
+  viewer:
+    subscriptions:
+      - 'users'
+      - 'chat' 
+      - 'cursor'
+      - 'deskshare'
+      - 'meetings'
+      - 'polls'
+      - 'presentations'
+      - 'shapes'
+      - 'slides'
+      - 'captions'
+      - 'breakouts'
+    methods:
+      - 'kickUser'
+      - 'listenOnlyToggle'
+      - 'userLogout'
+      - 'assignPresenter'
+      - 'setEmojiStatus'
+      - 'validateAuthToken'
+      - 'muteUser'
+      - 'unmuteUser'
+      - 'switchSlide'
+      - 'switchSlideMessage'
+      - 'publishVote'
+      - 'logClient'
+      - 'sendChat'
+      - 'switchSlideMessage'
+  moderator:
```

Comment:
To cover all the bases you'd basically need two sets of permissions. You've got guest (2.0), viewer, moderator, and then presenter is its own set of permissions because as you point out a person who's a viewer or moderator can be the presenter.

It also looks like the two lists are identical. Viewers shouldn't be able to kick or make someone the presenter for instance.